tsinfer/inference.py

```diff
@@ -1910,41 +1958,79 @@ def dask_find_path(
         site_indexes=None,
         sample_id_map=None,
     ):
-        t = time.time()
+        result = SampleMatcher.inner_find_path(
+            samples_slice,
+            tree_sequence_builder_wrapper.tsb,
+            data_path,
+            engine,
+            recombination,
+            mismatch,
+            precision,
+            extended_checks,
+            site_indexes,
+            sample_id_map,
+        )
+        # Pickle here rather than let dask deal with it so we can log sizes
+        return pickle.dumps(result)
+
+    def find_path_to_disk(self, samples_slice, output_path):
+        result = self.inner_find_path(
```

Comment:
named parameter would make this less error-prone

Reply:
Fixed